rfcs/README.md

```diff
@@ -0,0 +1,18 @@
+# CloudFormation Guard Request for Comments (RFCs)
+
+Most bug fixes, doc enhancements, and small, incremental improvements for CloudFormation Guard can be surfaced and resolved via GitHub issues and pull requests. However, any suggestions for substantial changes to the ruleset langauge or tooling should be documented and actioned upon via a request for comments (RFC). This will provide a standardized process for discussion major improvements to the tooling and ruleset language.  
```

Comment:
Main question I think will be more clearly defining the line between small improvements and more substantial changes, but it'll always be subjective and hopefully naturally becomes clearer over time